etc/kayobe/kolla.yml

```diff
@@ -353,7 +353,7 @@ kolla_build_blocks:
     {% if kolla_base_distro in ['centos', 'rocky'] %}
     RUN \
         if [ -f /etc/yum.repos.d.backup/repos.tar.gz ]; then \
-          tar -xzf /etc/yum.repos.d.backup/repos.tar.gz -C /etc/yum.repos.d && \
```

Comment:
We would hide a non-zero return code without &&. Is that expected?

Reply:
I was seeing this error, which I thought might be being caused by using the && here. (I'll admit my bash scripting knowledge is patchy)
```
INFO:kolla.common.utils.neutron-server:Step 7/8 : RUN     if [ -f /etc/yum.repos.d.backup/repos.tar.gz ]; then       tar -xzf /etc/yum.repos.d.backup/repos.tar.gz -C /etc/yum.repos.d &&       rm -rf /etc/yum.repos.d.backup/     fi &&     if grep -r 'http://pulp-server.internal.sms-cloud:8080/' /etc/yum.repos.d; then       echo "Found repository mirror in Yum repositories";       exit 1;     fi
    INFO:kolla.common.utils.neutron-server: ---> Running in f9498830e462
    INFO:kolla.common.utils.neutron-server:[91m/bin/sh: -c: line 2: syntax error: unexpected end of file

Reply:
I've restored the &&, I suspect the error is from missing semicolons due to this being pushed into an inline bash command